interpreter/cling/lib/Interpreter/TransactionPool.h

```diff
@@ -48,10 +47,9 @@ namespace cling {
 
     Transaction* takeTransaction(clang::Sema& S) {
       Transaction *T;
-      if (kDebugMode || m_Transactions.empty()) {
-        T = (Transaction*) ::operator new(sizeof(Transaction));
-        new(T) Transaction(S);
-      } else
+      if (kDebugMode || m_Transactions.empty())
+        T = new Transaction(S);
```

Comment:
Does that have noticeable performance implication?

Reply:
You mean the potential speed-up? I doubt it. (If you expected a degradation then you probably misread what the old code does ;-) )